controller/controllerLogOut.js

```diff
@@ -0,0 +1,21 @@
+import { ListUser } from "../schemas/listUserSchema";
+
+export const logOut = async (req, res) => {
+  try {
+    const authHeader = req.headers.authorization;
```

Comment:
Ten kontroler zanim zostanie uzyty to wykonasz juz authMiddleware. I dlatego nie musisz juz sprawdzac req.headers
`router.get("/logout", authMiddleware, logOut);`